packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -87,8 +87,11 @@ export class LitAuthClient {
     if (options?.litNodeClient) {
       this.litNodeClient = options?.litNodeClient;
     } else {
+      if (!options.litNetwork) {
+        console.warn(`litNetwork is required to create a LitNodeClient`);
+      }
       this.litNodeClient = new LitNodeClient({
-        litNetwork: 'cayenne',
+        litNetwork: options.litNetwork,
```

Comment:
I think we should still default it in case the user doesnt supply a network option. But maybe we want this to be a breaking change?

Reply:
Yea, this will still default to `cayenne` as this is set by default in `litNodeClient`. I agreed with Daryl that we should pass in by instance, and at the same time we need to prioritize addressing and patching any issues as soon as they arise. This PR can serve as a useful reference for future refactoring efforts..

Reply:
I agree, we should prioritize addressing/patching the issue... I just don't see why we can't do that by removing the offending implicit behaviour entirely.

I'm also confused about the content of this PR -- the current code doesn't even fix the problem; a console.warn() doesn't actually make the new argument required like the PR title implies this code does.

The PR description says:
`We should force to provide lit network when initialising lit auth client, and we shouldn't default to cayenne`
However, the current PR doesn't actually _force_ consumers to provide the lit network (it's optional), _and_ it still implictly defaults to `cayenne` in cases where they don't provide a litNetwork property.

The fact that we're even talking about this change this much IMO is a clear sign that we should just _let the consumer create the client instance and pass it in here_.  The current API is just extra mental overhead for people to grok, and adding a `litNetwork` property here is even _worse_.  What happens if I pass a `custom` lit node client configured for `cayenne`, but then I provide the _optional_ argument with `litNetwork` set to `manzano`?  As it turns out, I think that the new `litNetwork` argument will be _ignored_ if I provide a custom client, but how am I supposed to know that? As a consumer of this API I have no idea what will happen in that case without reading the source code.